src/dmd/backend/elfobj.d

```diff
@@ -3683,7 +3739,8 @@ else
                 buf.writeByte(0x83);
                 buf.writeByte(modregrm(0,7,5));
                 off += 3;
-                off += Obj_writerel(codseg, off, R_X86_64_GOTPCREL, symidx, -5);
+                const reltype = /*config.flags3 & CFG3pie ? R_X86_64_PC32 :*/ R_X86_64_GOTPCREL;
```

Comment:
Ditto.

Reply:
I'm not sure which of these is needed, left it in there for a clue if I do need the other one.